assets/js/atomic/blocks/product/add-to-cart/shared/add-to-cart-button.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+import Button from '@woocommerce/base-components/button';
+import { Icon, done as doneIcon } from '@woocommerce/icons';
+import { useState } from '@wordpress/element';
+import { useAddToCartFormContext } from '@woocommerce/base-context';
+
+/**
+ * Add to Cart Form Qty + Button Block Component.
```

Comment:
Is this comment accurate? This looks like just the CTA/button component, the quantity picker is elsewhere.